pygerduty/v2.py

```diff
@@ -132,24 +136,26 @@ def list(self, **kwargs):
                     'offset': offset,
                 })
                 this_paginated_result = self._list_no_pagination(**these_kwargs)
-                print this_paginated_result
+
                 if not this_paginated_result:
                     break
                 for item in this_paginated_result:
                     if item.id in seen_items:
                         continue
                     seen_items.add(item.id)
                     yield item
-                offset += len(this_paginated_result)
-                print len(this_paginated_result)
-                print "limit is: %s" % limit
-                print "offset is: %s" % offset
+                offset -= len(this_paginated_result)
```

Comment:
@gmjosack @rossdylan Can you both check the logic here? This change allowed `test_list_incidents_v2` to pass, but I am not sure if this is what was originally intended.